### PR TITLE
Add Right-to-Left support

### PR DESCRIPTION
**Pull request by ofekashery**
The beginning of Hebrew support 🥳

### CODE REVIEW

**Comment by AustinShalit** [comment]
This looks great! If it helps you at all (not sure if it will) take a look at the PR here: https://github.com/readthedocs/sphinx_rtd_theme/pull/986

**Comment by Daltz333** [comment]
A very pleasant surprise!

**Comment by ofekashery** [comment]
Thanks! I think it's ready.

By the way, it seems like something is broken with the [Hebrew docs](https://docs.wpilib.org/he/latest/) in production - it works for me locally.

**Comment by Daltz333** [comment]
Hebrew isn't enabled 😄

**Comment by Daltz333** [comment]
Something that would be nice, if things like admonitions (warnings/notes), hoverxref (hover tooltip content), and the extra things we have in the docs (code-tabs) could be RTL aligned.

**Comment by ofekashery** [comment]
The admonitions and code-tabs are already RTL aligned.
Do you have an example of a page that uses the ``hoverxref``? I didn't find it.

image
image

**Comment by Daltz333** [comment]
https://docs.wpilib.org/en/latest/docs/software/advanced-controls/introduction/tuning-pid-controller.html

**Comment by Daltz333** [comment]
@Kevin-OConnor what are your thoughts on the CSS styling and changes

**Comment by ofekashery** [comment]
Here is how it looks:
image

**Comment by AustinShalit** [comment]
I am not sure why this would look this way but does the Zayin look backward to you?

image

**Comment by ofekashery** [comment]
Actually, if I focus on that, it looks a bit backward, but it's part of Heebo's typography (We use Heebo as our primary font at _FIRST_ Israel).

**Comment by AustinShalit** [comment]
Ah ok. I don't want to go messing with the typography.

Someone pointed out to me that the `C++` gets converted to `++C` in the image you posted. I think this is not standard?

**Comment by ofekashery** [comment]
It's not expected but unfortunately we don't have much to do with it.
It's generally caused because the characters being written right-to-left, and the ``++`` are locale-free characters, so they are rendered after the ``C``. If there was another English char after the ``++`` it would be okay.

A possible solution is to wrap it in a ``span`` with left-to-right direction (``<span dir="ltr">C++</span>``), but I'm not sure how to do that on the translation side.

This issue seems to also exist in [other sites](https://campus.gov.il/course/course-v1-basmach-pc264/) that write ``C++`` in a Hebrew sentence...

**Comment by Daltz333** [comment]
Ah. I'm gonna spend a bit of time testing locally tomorrow, so expect a review then

File: source/_static/css/frc-rtl.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+.rst-versions .rst-current-version {
+  text-align: left;
+}
+.rst-versions .rst-current-version .fa-book, .rst-versions .rst-current-version .icon-book {
+  float: right;
+}
+.rst-other-versions {
+  text-align: right;
+}
```
Comment (AustinShalit) [inline source/_static/css/frc-rtl.css:75]:
```suggestion
}

```

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**